packages/elements/src/combo-box/__test__/combo-box.filter.test.js

```diff
@@ -40,5 +42,37 @@ describe('combo-box/Filter', function () {
       expect(el.query).to.equal(textInput, 'Query should be the same as input text: "Aland Islands"');
       await expect(el).shadowDom.to.equalSnapshot(snapshotIgnore);
     });
+
+    it('Should be able to use custom filter function', async function () {
```

Comment:
This test case doesn't available v6. Please port this test case to v6.

Reply:
Another good catch! Added.